backend/apps/search/migrations/0004_searchdocument_add_description_tags_typename.py

```
@@ -0,0 +1,31 @@
+        migrations.AddField(
+            model_name="searchdocument",
+            name="tags",
+            field=models.CharField(blank=True, default="", max_length=500),
+        ),
+        # Denormalized content type name for O(1) API-level filtering
+        migrations.AddField(
+            model_name="searchdocument",
+            name="content_type_name",
+            field=models.CharField(blank=True, default="", max_length=100),
+        ),
+    ]
```

Ruff (0.15.20)

[warning] Mutable default value for class attribute

(RUF012)